src/color.cc

```diff
@@ -21,8 +21,10 @@
 #include "wabt/common.h"
 
 #if _WIN32
+#if HAVE_WIN32_VT100
```

Comment:
I don't think you need both here.. wouldn't `HAVE_WIN32_VT100` be enough on its own?

Reply:
Yeah true. I can make this change in a new PR

Reply:
In https://github.com/WebAssembly/wabt/pull/2231